doc/source/multivisit.rst

```diff
@@ -3,4 +3,435 @@
+
+Additionally, the function `run_mvm_processing()` has the ability to enable an additional attempt to align all the
+input exposures to the latest astrometric catalog, as well as limit the size of the output mosaics.
+Full details of these parameters are available in the discussion of the MVM API.
+
+
+Define SkyCell Layers
+----------------------
+SkyCells define the WCS that will be used for all the observations for any given region on the sky.  However, it doesn't
+make sense to create a single image from all the exposures due to differences in the detectors, pixel sizes, PSFs and
+filters used for the observations.  It can even be argued that observations taken too far apart in time should also not
+be combined, or observations taken with dramatic differences in exposure time should not be combined.  As a result,
+the concept of a SkyCell 'layer' was implemented to organize all the exposures for a SkyCell into sets of exposures which
+can be combined to create useful, and hopefully scientifically interesting, mosaics.
+
+The most basic definition for a layer organizes the exposures based on the following criteria:
+
+  * instrument
+  * detector
+  * filter
+
+Using the renamed input files, the MVM processing code organizes all the exposures based on these criteria to identify
+what layers could be generated from all the inputs.  For example, a SkyCell with ACS/WFC3 F814W exposures and
+WFC3/UVIS F606W exposures would result in 2 SkyCell layers being defined; namely, one for each set of exposures.
+
+.. note:: Observations taken with a spectroscopic element, like grisms or prisms, will not be used to define SkyCell layers.
+
+
+Determine Layers to Process
+----------------------------
+Specifying what input exposures need to be used to create a SkyCell layer mosaic serves as a critical feature of the
+input file.  Input files simply listing filenames indicate that ALL exposures specified should be used to create MVM
+products.  However, the more descriptive `poller file` CSV format input file provides the ability to limit the processing
+to only new exposures, while treating already archived versions of the MVM products for all the other SkyCell layers as
+fully updated and not in need of any further processing.  This control comes from the entry that specifies 'NEW' or 'OLD',
+with only those layers with at least 1 'NEW' entry getting defined for processing.
+
+
+Create SkyCell Mosaics
+----------------------
+The interpretation of the input files performed to this point has resulted in the definition of the image products that
+need to be created by combining the exposures specified for processing.  These mosaics get generated by drizzling the
+input exposures onto the WCS defined for the SkyCell, creating a separate mosaic for each layer from all the exposures
+with the same filter/detector/instrument configuration.
+
+The drizzle parameters used to create these products get determined based on the average number of exposures for all the
```

Comment:
Change "get determined" to "are determined".

Reply:
Changed in 241a672